app/views/confidentialInformation.scala.html

```diff
@@ -0,0 +1,45 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+@import uk.gov.hmrc.play.views.html._
+@import controllers.routes._
+@import models.{Mode, ConfidentialInformation}
+@import views.ViewUtils._
+
+@(appConfig: FrontendAppConfig, form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = s"${errorPrefix(form)} ${messages("confidentialInformation.title")}",
+    appConfig = appConfig,
+    bodyClasses = None) {
+
+    @helpers.form(action = ConfidentialInformationController.onSubmit(mode), 'autoComplete -> "off") {
+
+        @components.back_link()
+
+        @components.error_summary(form.errors)
+
+        @components.heading("confidentialInformation.heading")
+
+        @components.input_textarea(
+            field = form("field1"),
```

Comment:
I think it's important we change the `field1`, `field2`s as we're going along, otherwise it becomes quite a large piece of tech debt to fix later on

Reply:
ALL text input field ID's are auto generated by scaffold like this. Scaffold only allows you to change radio button ID's when generating pages. These can be changed, but will need to be changed in several places. It already is tech debt, worth raising a bug to fix and address these.